omnibus/package-scripts/agent-rpm/posttrans

```diff
@@ -75,6 +75,35 @@ if [ "$INSTALL_SELINUX_POLICY" = "true" ]; then
 fi
 
 
+# Set the installation information if not already present;
```

Comment:
Looks like the same code is duplicated twice here?

Reply:
Oh, good catch!